token/cli/src/main.rs

```diff
@@ -631,32 +666,29 @@ async fn command_authorize(
             auth_str,
             previous_authority
                 .map(|pubkey| pubkey.to_string())
-                .unwrap_or_else(|| "disabled".to_string()),
+                .unwrap_or_else(|| if config.sign_only {
+                    "unknown".to_string()
```

Comment:
nit: I don't think it'll be possible to get to this line since it's always `None` in sign only mode

Reply:
unless im misunderstanding which line, it should _always_ get this line in sign only. its to clear up a tiny display inaccuracy: in normal mode, `None` means there is no authority, but in sign only, we dont know

Reply:
Ah sorry, yes you're absolutely right, I reviewed this a little too quickly, and didn't see `unwrap_or_else`, and thought it was happening on `map`.  All good!